test/unit/sql/update.js

```diff
@@ -34,5 +34,28 @@ describe(Support.getTestDialectTeaser('SQL'), function() {
       });
     });
 
+    
+    it('Works with limit', function () {
+      const User = Support.sequelize.define('User', {
+        username: {
+          type: DataTypes.STRING
+        },
+        userId: {
+          type: DataTypes.INTEGER
+        }
+      }, {
+        timestamps: false
+      });
+
+      expectsql(sql.updateQuery(User.tableName, { username: 'new.username' }, { username: 'username' }, { limit: 1 }),
+      {
+        mssql: "UPDATE TOP(1) [Users] SET [username]=N'new.username' OUTPUT INSERTED.* WHERE [username] = N'username'",
+        postgres: "UPDATE \"Users\" SET \"username\"=\'new.username\' WHERE \"username\" = \'username\'",
+        sqlite: "UPDATE `Users` SET `username`='new.username' WHERE `username` = 'username' LIMIT 1",
+        default: "UPDATE `Users` SET `username`=\'new.username\' WHERE `username` = \'username\' LIMIT 1"
```

Comment:
`default: "UPDATE [Users] SET [username]='new.username' WHERE [username] = 'username' LIMIT 1",`

[ and ] are then replaced with the quotes for that specific dialect (or kept as is for mssql :) )